md/data-handling-overview.md

```diff
@@ -1,5 +1,7 @@
 # Data handling overview
 
+How to effectively use the different kinds of data in the Bonita application
```

Comment:
We usually talk about Bonita Platform in the rest of the documentation. So I would change the end of the sentence: "How to effectively use the different kinds of data available within Bonita Platform"

Reply:
I agree it should be consistent with other docs